src/Data/Profunctor/Unsafe.hs

```diff
@@ -189,6 +199,10 @@ instance Profunctor Tagged where
   Tagged s .# _ = Tagged s
   {-# INLINE (.#) #-}
 
+-- TODO: orphan instance
```

Comment:
Let's add it to `base` and `base-compat` instead?

Reply:
Yes, it was kinda unexpected for me to see absence of the `Functor` instance for `Kleisli`. The definition is straightforward. Should I open new issue here?

* https://gitlab.haskell.org/ghc/ghc/issues

Some relevant discussion:

* https://stackoverflow.com/questions/18925045/why-there-isnt-a-functor-instance-for-kleisli-in-control-arrow

Reply:
The usual process is to propose the instance on https://mail.haskell.org/mailman/listinfo/libraries and submit a patch if there is agreement.

Reply:
The instances for `base` are in [`base-orphans`](https://hackage.haskell.org/package/base-orphans)